server/src/main/java/org/opensearch/rest/action/admin/cluster/RestClusterStatsAction.java

```diff
@@ -68,6 +68,8 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
         clusterStatsRequest.timeout(request.param("timeout"));
         clusterStatsRequest.setIncludeDiscoveryNodes(false);
         clusterStatsRequest.useAggregatedNodeLevelResponses(true);
+        clusterStatsRequest.setIncludeMappingStats(request.paramAsBoolean("include_mapping_stats", true));
+        clusterStatsRequest.setIncludeAnalysisStats(request.paramAsBoolean("include_analysis_stats", true));
```

Comment:
Have we discussed these param names anywhere?

Reply:
No there was no separate discussion for these param names. It can be part of this PR itself? open for suggestions.